src/endpoints/scraper.ts

```diff
@@ -0,0 +1,132 @@
+//#region > Imports
+//> Interfaces
+// Contains the interface for the Apollo endpoint and options
+import { ScraperEndpoint, Options } from "./index";
+//#endregion
+
+//#region > Classes
+/** @class A endpoint to fetch page DOM */
+class Scraper implements ScraperEndpoint {
+  //> Fields
+  headers: object = {
+    accept: "application/json, text/plain, */*",
+  };
+
+  desc: string = "A endpoint to fetch page DOM";
+  proxy: string = "https://cors.snek.at/";
+
+  /**
+   * @constructor
```

Comment:
Should have some sort of comment.

Initializes a Scraper.

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3